packages/fxa-settings/src/components/App/HeaderContinueTo/index.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Localized } from '@fluent/react';
+
+export type HeaderContinueToProps = {
+  prefixDataTestId: string;
+  prefixFtlId: string;
+  headingText: string;
+  serviceName?: string;
+};
+
+const HeaderContinueTo = ({
+  prefixDataTestId,
+  prefixFtlId,
+  headingText,
+  serviceName,
+}: HeaderContinueToProps & RouteComponentProps) => {
```

Comment:
This shouldn't need `& RouteComponentProps`, that's only necessary when the component is used as a route/page.